automatic/pencil/tools/chocolateyInstall.ps1

```diff
@@ -5,8 +5,7 @@ $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
 $packageArgs = @{
   packageName    = $Env:ChocolateyPackageName
   fileType       = 'exe'
-  file           = Get-Item $toolsPath\*-i386.exe
-  file64         = Get-Item $toolsPath\*_64.exe
+  file           = Get-Item $toolsPath\Pencil-*.exe
```

Comment:
We avoid doing this now and all for the file name to be explicitly used.

Reply:
I've fixed that up